Presentation.Web/Tests/01_Parallel/It-Contract/LocalAdmin.ItContract.UiCustomization.e2e.spec.ts

```diff
@@ -23,23 +30,66 @@ describe("Local admin is able customize the IT-Contract UI", () => {
 
     var localContractPath: string | null = null;
 
-    it("Disabling Tabs/fields will hide the tabs/fields on the IT-Contract details page", () => {
-        var contractName = createName("contractName");
-        var orgName = createName("orgName");
+    it("Disabling tabs will hide the tabs on the IT-Contract details page", () => {
 
-        return loginHelper.loginAsGlobalAdmin()
-            .then(() => OrgHelper.createOrg(orgName))
-            .then(() => OrgHelper.changeOrg(orgName))
-            .then(() => ContractHelper.createContract(contractName))
-            .then(() => navigation.getPage("/#/global-admin/local-admins"))
-            .then(() => Select2Helper.select(orgName, "s2id_selectOrg"))
-            .then(() => Select2Helper.select(loginHelper.getGlobalAdminCredentials().username, "selectUser"))
+        return setupUser()
             .then(() => testTabCustomization(contractName, "ItContracts.contractRoles", ContractNavigationSrefs.contractRolesSref))
-            .then(() => testTabCustomization(contractName, "ItContracts.advice", ContractNavigationSrefs.adviceSref))
+            .then(() => testTabCustomization(contractName, "ItContracts.advice", ContractNavigationSrefs.adviceSref));
+    });
+
+    it("Disabling fields will hide the fields on the IT-Contract front page", () => {
+
+        return setupUser()
```

Comment:
I tried adding the "setupUser" part also to the "beforeAll" function, but it wouldn't work, and every test would fail.

I'm guessing there is a simple fix to solve this, that I'm missing

Reply:
I think it has to do with timeout on the beforeall method. 
I also think we should isolate each test run on its own org, so in stead of beforeall we should do a beforEach